otter/integration/tests/test_convergence.py

```diff
@@ -208,6 +319,7 @@ def delete_server_by_id(i):
                 .addCallback(lambda _: rcs)
             )
 
+        self.deleted_server_ids = ids
```

Comment:
I already have a `removed_ids` attribute for this purpose.  Have you forward-merged from master and re-pushed the PR?